IpmiFeaturePkg/Library/IpmiCommandLib/IpmiCommandLibNetFnApp.c

```diff
@@ -11,6 +11,7 @@
 #include <Library/IpmiBaseLib.h>
 
 #include <IndustryStandard/Ipmi.h>

```

Comment:
nit, can remove Ipmi.h, might be best to normalize using just IpmiFeature to make it the one-stop shop for definitions